docs/release_notes/0.152.0.md

```diff
@@ -5,12 +5,9 @@
 - Add support for Tel Aviv (il-central-1) (#6899)
 - Add option to print specific events when running `eksctl describe-stacks` (#6851)
 
-## 🐛 Bug Fixes
-
-- Fix PR Rebase workflow (#6903)
-
 ## 🧰 Maintenance
 
+- Remove PR rebase workflow (#6934)
```

Comment:
This will have no effect on the release notes, so we should also update that.

Reply:
Yes, my initial intention was to update the release branch as-well.

Reply:
Please remove changes here, it's not required.